Reminder-App/Reminder-App/Views/ReminderListCell.swift

```diff
@@ -11,29 +11,30 @@ class ReminderListCell: UITableViewCell {
     
     typealias DoneButtonAction = () -> Void
     
-    //MARK:- Outlets
+    //MARK: Outlets
```

Comment:
Your MARK is still not consistent. earlier you have a space after second slash but here you don't have.